### PR TITLE
Cherry-pick #11982 to 7.1: Change type from scaled_float to long and add format (#11982)

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
Cherry-pick of PR #11982 to 7.1 branch. Original message:

I'm gonna try this one more time... A lot of `scaled_float` here in aws module does not need to be `scaled_float`, a `long` type should be sufficient here. Also, when I first started aws module, I didn't know about the defined `format` in Elasticsearch. This PR also adds `format: bytes` and `format: duration` into `fields.yml`.

* Change type from scaled_float to long and add format

(cherry picked from commit 1b2613eb6dc3b81bf6fd3c5141b4f9f7976953dd)

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -20,6 +20,11 @@ https://github.com/elastic/beats/compare/v7.0.0...7.1[Check the HEAD diff]
 
 *Metricbeat*
 
+- Add new option `OpMultiplyBuckets` to scale histogram buckets to avoid decimal points in final events {pull}10994[10994]
+- Change cloud.provider from ec2 to aws and from gce to gcp in add_cloud_metadata to align with ECS. {issue}10775[10775] {pull}11687[11687]
+- system/raid metricset now uses /sys/block instead of /proc/mdstat for data. {pull}11613[11613]
+- Change some field type from scaled_float to long in aws module. {pull}11982[11982]
```
Comment (exekias) [inline CHANGELOG.next.asciidoc]: you will need to fix this, it happens some time with backports

Comment (kaiyan-sheng) [reply CHANGELOG.next.asciidoc]: @exekias yeah and the fields.yml, working on it :-)

File: metricbeat/docs/fields.asciidoc [hunk 2]
```diff
@@ -1261,7 +1284,13 @@ timecheck round
 *`ceph.cluster_health.timechecks.round.status`*::
 +
 --
+<<<<<<< HEAD
```
Comment (exekias) [inline metricbeat/docs/fields.asciidoc]: it seems some merge errors happened, you will need to run make update

Comment (kaiyan-sheng) [reply metricbeat/docs/fields.asciidoc]: yep, currently waiting for make update to finish 👍

**Pull request merged**